### PR TITLE
Fixed #35072 -- Corrected Field.choices description in models topic.

### PR DESCRIPTION
**Pull request by 1zzowiebeha**
Fixed choices description in docs: [ticket](https://code.djangoproject.com/ticket/35072)

### CODE REVIEW

File: docs/topics/db/models.txt [hunk 1]
```diff
@@ -202,7 +204,7 @@ ones:
         >>> p.get_shirt_size_display()
         'Large'
 
-    You can also use enumeration classes to define ``choices`` in a concise
+    Here's how you can use enumeration classes to define ``choices`` in a concise
```
Comment (felixxm) [inline docs/topics/db/models.txt:207]: This change is not required, IMO. Please revert.

File: docs/topics/db/models.txt [hunk 2]
```diff
@@ -157,9 +157,11 @@ ones:
...
 
 :attr:`~Field.choices`
-    A :term:`sequence` of 2-tuples to use as choices for this field. If this
-    is given, the default form widget will be a select box instead of the
-    standard text field and will limit choices to the choices given.
+    A :term:`sequence` of 2-value tuples, a :term:`mapping`, an :ref:`enumeration type <field-choices-enum-types>`,
+    or a parameter-less :term:`callable` which returns one of the previous formats
+    may be passed as choices for this field. If this is given, the default form widget
+    will be a select box instead of the standard text field,
+    and the widget will limit its choices to those that are given.
```
Comment (felixxm) [inline docs/topics/db/models.txt:164]: Wrap at 79 chars and add a missing `versionchanged` annotation:
```
.. versionchanged:: 5.0

    Support for mappings and callables was added.

```

File: docs/topics/db/models.txt [hunk 3]
```diff
@@ -157,9 +157,11 @@ ones:
     <Field.blank>`, the field will be required.
 
 :attr:`~Field.choices`
-    A :term:`sequence` of 2-tuples to use as choices for this field. If this
-    is given, the default form widget will be a select box instead of the
-    standard text field and will limit choices to the choices given.
+    A :term:`sequence` of 2-value tuples, a :term:`mapping`, an :ref:`enumeration type <field-choices-enum-types>`,
+    or a parameter-less :term:`callable` which returns one of the previous formats
```
Comment (felixxm) [inline docs/topics/db/models.txt:161]: I'd leave only `a callable` without no further details:

```a parameter-less :term:`callable` which returns one of the previous formats``` -> ```a :term:`callable` ```

File: docs/topics/db/models.txt [hunk 4]
```diff
@@ -157,9 +157,11 @@ ones:
...
+    may be passed as choices for this field. If this is given, the default form widget
```
Comment (felixxm) [inline docs/topics/db/models.txt:162]: I'd keep the current wording
```suggestion
    to use as choices for this field. If this is given, the default form widget
```

**Review by felixxm — COMMENTED** [review]
@1zzowiebeha Thanks :+1: I left comments.

**Comment by felixxm** [comment]
@1zzowiebeha Thanks :+1: Can you allow edits from maintainers? I want to push small final edits.

image

**Comment by 1zzowiebeha** [comment]
image
I think I have the option checked to allow edits. Is there another place to give you permissions?

**Comment by felixxm** [comment]
> I think I have the option checked to allow edits. Is there another place to give you permissions?

Sorry, my fault.

Welcome aboard :boat:

**Pull request merged**